theano/gpuarray/dnn_gw.c

```diff
@@ -154,8 +154,8 @@ APPLY_SPECIFIC(conv_gw)(PyGpuArrayObject *input, PyGpuArrayObject *output,
         return 1;
       }
 
-      // Guess 4Mb if the info is not available
-      if (free == 0) free = 4 * 1024 * 1024;
+      // Guess 1Gb if the info is not available
+      if (free == 0) free = 1 * 1024 * 1024 * 1024;
```

Comment:
4MB is very low for common nets - I have seen larger than 1G workspaces required.